app/src/main/java/org/thoughtcrime/securesms/onboarding/pickname/PickDisplayName.kt

```diff
@@ -0,0 +1,69 @@
+package org.thoughtcrime.securesms.onboarding.pickname
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import network.loki.messenger.R
+import org.thoughtcrime.securesms.ui.PreviewTheme
+import org.thoughtcrime.securesms.ui.base
+import org.thoughtcrime.securesms.ui.components.OutlineButton
+import org.thoughtcrime.securesms.ui.components.SessionOutlinedTextField
+import org.thoughtcrime.securesms.ui.contentDescription
+import org.thoughtcrime.securesms.ui.h4
+
+@Preview
+@Composable
+fun PreviewDisplayName() {
+    PreviewTheme {
+        DisplayName(State())
+    }
+}
+
+@Composable
+fun DisplayName(state: State, onChange: (String) -> Unit = {}, onContinue: () -> Unit = {}) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(20.dp),
+        modifier = Modifier
+            .padding(horizontal = 50.dp)
+            .padding(bottom = 12.dp)
```

Comment:
Haven't we standardised all these?
Also this should be similar to, for example, the recovery password screen, which is identical, yet they are defined differently.

Reply:
yeah... I've improved this. Some of the dimensions are hard to read off of figma, but I've noticed some more commonalities... done